<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRsearchclass</class>
 <widget class="QDialog" name="CRsearchclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check Classroom</string>
  </property>
  <widget class="QFrame" name="frame4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Room 402</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 60, 0);
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame6">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>390</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Room 404</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 60, 0);
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>                             Dashboard</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame9">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>540</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Room 203</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 60, 0);
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>240</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Room 310</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 60, 0);
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Room 302</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 60, 0);
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame8">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Room 202</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 60, 0);
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>140</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>240</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Room 304</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 60, 0);
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>540</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Room 201</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 60, 0);
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Room 403</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 60, 0);
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>0</y>
     <width>1691</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:bold;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Logged in as CR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonback">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 253);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 110, 253);</string>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>frame4</zorder>
  <zorder>frame6</zorder>
  <zorder>label_2</zorder>
  <zorder>frame9</zorder>
  <zorder>frame3</zorder>
  <zorder>frame1</zorder>
  <zorder>frame8</zorder>
  <zorder>labelTime</zorder>
  <zorder>frame2</zorder>
  <zorder>frame7</zorder>
  <zorder>frame5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
